peekingduck/runner.py

```diff
@@ -90,11 +91,14 @@ def __init__(  # pylint: disable=too-many-arguments
             self.num_iter = num_iter
             self.logger.info(f"Run pipeline for {num_iter} iterations")
 
-    def run(self) -> None:
+    def run(self) -> None:  # pylint: disable=too-many-branches
         """execute single or continuous inference"""
         num_iter = 0
         while not self.pipeline.terminate:
             for node in self.pipeline.nodes:
+                if num_iter == 0:  # report init times at first iteration
+                    self.logger.info(f"Initialising {node.name}...")
```

Comment:
Can we remove this logging statement? It's already logged in declarative loader, now this adds another logging statement in runner.
```
2022-01-13 14:29:13 peekingduck.declarative_loader  INFO:  Initialising input.live node... 
2022-01-13 14:29:14 peekingduck.pipeline.nodes.input.live  INFO:  Device resolution used: 1280 by 720 
2022-01-13 14:29:14 peekingduck.declarative_loader  INFO:  Initialising output.screen node... 
2022-01-13 14:29:14 peekingduck.cli  INFO:  Startup time = 1.21 sec 
2022-01-13 14:29:14 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.input.live... 
2022-01-13 14:29:14 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.input.live init time = 0.02 sec 
2022-01-13 14:29:14 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.output.screen... 
2022-01-13 14:29:14 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.output.screen init time = 0.06 sec 
```

Reply:
Have changed the log level for timing messages from `info` to `debug`.